### PR TITLE
editorial nits

### PR DESCRIPTION
**Pull request by moonshiner**
Some minor editorial nits to the text

### CODE REVIEW

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ A catalog zone is a DNS zone whose contents are specially crafted. Its records p
 
 Catalog consumers SHOULD ignore any RR in the catalog zone which is meaningless or useless to the implementation.
 
-Authoritative servers may be preconfigured with multiple catalog zones, each associated with a different set of configurations.
+Authoritative servers may be pre configured with multiple catalog zones, each associated with a different set of configurations.
```
Comment (jsoref) [inline draft-ietf-dnsop-dns-catalog-zones.md:156]: This seems wrong

**Comment by moonshiner** [comment]
argh - I left some in that I thought I had removed. 

perhaps instead of saying "all associated state" that "all the associated state" sounds more straight forward. 

and It seems like "preconfigure" and "pre-configure" are similar so just leave as is for the RFC editor?

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 1 above)
Comment (wtoorop) [inline draft-ietf-dnsop-dns-catalog-zones.md:156]:
```suggestion
Authoritative servers may be preconfigured with multiple catalog zones, each associated with a different set of configurations.
```

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 2]
```diff
@@ -284,23 +284,23 @@ coo.<unique-N>.zones.$OLDCATZ 0 IN PTR $NEWCATZ
 The PTR RRset MUST consist of a single PTR record.
 More than one record in the RRset denotes a broken catalog zone which MUST NOT be processed (see (#generalrequirements)).
 
-When a consumer of catalog zone `$OLDCATZ` receives an update which adds or changes a `coo` property for a member zone in `$OLDCATZ`, it does *not* migrate the member zone immediately.
+When a consumer of a catalog zone `$OLDCATZ` receives an update which adds or changes a `coo` property for a member zone in `$OLDCATZ`, it does *not* migrate the member zone immediately.
 The migration has to wait for an update of `$NEWCATZ`. in which the member zone is present. The consumer MUST verify, before the actual migration, that `coo` property pointing to `$NEWCATZ` is still present in `$OLDCATZ`.
 
-Unless the member node label (i.e. `<unique-N>`) for the member is the same in `$NEWCATZ`, all associated state for a just migrated zone MUST be reset (see (#zonereset)).
+Unless the member node label (i.e. `<unique-N>`) for the member is the same in `$NEWCATZ`, all associated states for a just migrated zone MUST be reset (see (#zonereset)).
```
Comment (wtoorop) [inline draft-ietf-dnsop-dns-catalog-zones.md:290]:
```suggestion
Unless the member node label (i.e. `<unique-N>`) for the member is the same in `$NEWCATZ`, all associated state for a just migrated zone MUST be reset (see (#zonereset)).
```

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 3]
```diff
@@ -489,12 +489,12 @@ It is RECOMMENDED to transfer catalog zones confidentially [@!RFC9103].
...
 be operated by different administrators.  The secondary nameservers may be
-configured as catalog consumer to synchronize catalog zones from the primary, but the primary's
+configured as a catalog consumer to synchronize catalog zones from the primary, but the primary's
 administrators may not have any administrative access to the secondaries.
 
 Administrative control over what zones are served from the configured name servers shifts completely from the server operator (consumer) to the "owner" (producer) of the catalog zone content.
 
-With migration of member zones between catalogs using the `coo` property, it is possible for the owner of the target catalog (i.e. `$NEWCATZ`) to take over all associated state with the zone from the original owner (i.e. `$OLDCATZ`) by maintaining the same member node label (i.e. `<unique-N>`).
+With migration of member zones between catalogs using the `coo` property, it is possible for the owner of the target catalog (i.e. `$NEWCATZ`) to take over all associated states with the zone from the original owner (i.e. `$OLDCATZ`) by maintaining the same member node label (i.e. `<unique-N>`).
```
Comment (wtoorop) [inline draft-ietf-dnsop-dns-catalog-zones.md:497]:
```suggestion
With migration of member zones between catalogs using the `coo` property, it is possible for the owner of the target catalog (i.e. `$NEWCATZ`) to take over all associated state with the zone from the original owner (i.e. `$OLDCATZ`) by maintaining the same member node label (i.e. `<unique-N>`).
```

**Pull request merged**